org.eclipse.m2e.rcptt.tests/pom.xml

```diff
@@ -11,15 +12,32 @@
 	<artifactId>org.eclipse.m2e.rcptt.tests</artifactId>
 	<packaging>rcpttTest</packaging>
 	<name>M2E - RCPTT Integration tests</name>
-	
+
 	<pluginRepositories>
 		<pluginRepository>
 			<id>rcptt</id>
 			<name>Eclipse RCPTT repository</name>
 			<url>https://repo.eclipse.org/content/repositories/rcptt-releases/</url>
 		</pluginRepository>
-	</pluginRepositories> 
-	
+	</pluginRepositories>
+
+	<properties>
+		<product-extension>tar.gz</product-extension>
+	</properties>
+	<profiles>
+		<profile>
+			<id>windows</id>
+			<activation>
+				<os>
+					<family>windows</family>
+				</os>
+			</activation>
+			<properties>
+				<product-extension>zip</product-extension>
+			</properties>
+		</profile>
+	</profiles>
```

Comment:
@ptziegler do you know a more elegant solution than this one to adapt the AUT to the different file-extensions used for the archives?

Reply:
I'm not aware of any way RCPTT can automatically detect the type of archive. Though I wonder if it would be less of a hassle to simply rewrite the test to use e.g. SWTBot...

Reply:
I think for our purpose it would be okay to always use `tar.gz` in all cases if it helps (we don't offer the products a user downloads where it would be a bit inconvenient).

Reply:
> I'm not aware of any way RCPTT can automatically detect the type of archive. Though I wonder if it would be less of a hassle to simply rewrite the test to use e.g. SWTBot...

I have no clue about this. I have neither worked with RCPTT nor with SWTBot yet, so I fully trust your assessment on this topic.

> I think for our purpose it would be okay to always use `tar.gz` in all cases if it helps (we don't offer the products a user downloads where it would be a bit inconvenient).

Yes I thought about this too, but was hoping there might be a more elegant solution.
But lets to it this way for now.

Reply:
> I have no clue about this. I have neither worked with RCPTT nor with SWTBot yet, so I fully trust your assessment on this topic.

The advantage of SWTBot is that it's executed via JUnit, so you don't really have to deal with file extensions, let alone the assembled product.